docroot/modules/humsci/hs_blocks/hs_blocks.libraries.yml

```diff
@@ -3,3 +3,8 @@ login.admin:
   css:
     base:
       css/hs_blocks.login.admin.css: {}
+block_group:
```

Comment:
Everything else is called group_block, why change now?

Reply:
just checking if you're paying attention